.travis-build.sh

```diff
@@ -71,13 +75,13 @@ if [[ "${GROUP}" == "downstream" || "${GROUP}" == "all" ]]; then
   ##  * daikon-typecheck: (takes 2 hours)
 
   # checker-framework-inference: 18 minutes
-  (cd .. && git clone --depth 1 https://github.com/typetools/checker-framework-inference.git)
+  (cd .. && git clone --depth 1 https://github.com/${SLUGOWNER}/checker-framework-inference.git)
```

Comment:
Thanks for the comment! Yes, adapted it to do the same fall-back as in .travis-build-without-tests.sh.